dev-java/commons-text/commons-text-1.11.0.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Skeleton command:
+# java-ebuilder --generate-ebuild --workdir . --pom pom.xml
+# --download-uri mirror://apache//commons/text/source/commons-text-1.10.0-src.tar.gz
+# --slot 0 --keywords "~amd64 ~ppc64 ~x86" --ebuild commons-text-1.10.0.ebuild
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source"
+MAVEN_ID="org.apache.commons:commons-text:${PVR}"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Apache Commons Text is a library focused on algorithms working on strings"
+HOMEPAGE="https://commons.apache.org/proper/commons-text/"
+SRC_URI="https://archive.apache.org/dist/commons/text/source/"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="amd64 ppc64 x86"
```

Comment:
Actually, stable keywords in overlays is a bad practice. I hope, you wouldn't mind if I'll change them to `~`.

Reply:
No problem at all, I hadn't clocked that when I copied `dev-java//commons-1.10.0.ebuild`, will pay more attention in the future.

Thanks for merging so quickly :+1: